dev/release/utils-prepare.sh

```diff
@@ -155,4 +155,10 @@ update_versions() {
   find . -name "*.bak" -exec rm {} \;
   git add .
   popd
+
+  pushd "${ARROW_DIR}"
+  python "./ci/scripts/util_update_docs_versions.py" . "${version}" "${r_version}"
```

Comment:
We should ensure using Python 3 not Python 2 because Python2's `json` puts a trailing space after comma: https://bugs.python.org/issue16333

```suggestion
  ${PYTHON:-python3} "./ci/scripts/util_update_docs_versions.py" . "${version}" "${r_version}"
```